features/imagebitmap.yml

```diff
@@ -0,0 +1,21 @@
+name: ImageBitmap
+description: The `ImageBitmap` interface represents a bitmap image which can be created from a variety of source objects. It can be drawn to a <canvas> without undue latency, and provides an asynchronous and resource efficient pathway to prepare textures for rendering in WebGL.
```

Comment:
OK, making a big rewrite suggestion here, aiming for a few things:

- `createImageBitmap()` is the entry point we ought to name first
- "variety of source objects" — let's use some examples instead
- "undue latency" — this sounds kinda lawyerly and I think we can do better by comparing it to higher latency things (I know this point and the one above is from the docs/specs, not assigning blame here)

That said, I wasn't able to preserve the stuff about WebGL. 

```suggestion
description: The `createImageBitmap()` global method creates an `ImageBitmap` object from a source such as an image, SVG, blob, or canvas. An `ImageBitmap` object represents pixel data that can be drawn to a canvas with lower latency than other types, such as `ImageData`.
```